docs/pages/management/guides/fluentd.mdx

```diff
@@ -38,40 +38,61 @@ from Teleport's events API, and forwards them to Fluentd.
   ```
   </TabItem>
 
-  <TabItem label="MacOS">
+  <TabItem label="macOS">
+
   ```code
   $ curl -L -O https://get.gravitational.com/teleport-event-handler-v(=teleport.plugin.version=)-darwin-amd64-bin.tar.gz
-  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-darwin-amd64-bin.tar.gz
   ```
+
+  We currently only build the event handler plugin for amd64 machines. If your
+  macOS machine uses Apple silicon, you will need to [install
+  Rosetta](https://support.apple.com/en-us/HT211861) before you can run the
+  event handler plugin. You can also build from source.
   </TabItem>
   <TabItem label="Docker">
   ```code
   $ docker pull public.ecr.aws/gravitational/teleport-plugin-event-handler:(=teleport.plugin.version=)
   ```
   </TabItem>
+  <TabItem label="From Source">
+  Ensure that you have Docker installed and running.
```

Comment:
The `make` target described below uses it. Since this is a backport, and we've already merged the original branch, we will need to make any new changes in another PR.